### PR TITLE
NMS-14658: Missing reference to logmsg dest component

### PR DESCRIPTION
**Pull request by Bonrob2**
NMS-14658: Missing reference to logmsg dest component - added information about the dest component to the events definition documentation.

### All Contributors

* [ ] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14658

### CODE REVIEW

**Review by mmahacek — APPROVED** [review]

File: docs/modules/operation/pages/deep-dive/events/event-definition.adoc [hunk 1]
```diff
@@ -27,8 +27,8 @@ Description::
     A long-form description of all instances of this event.
     You can use HTML formatting to format the body of the event text.
 Log Message::
-    A long-form log message describing this event, optionally including expansions of fields and parameters so that the value is tailored to the event at hand.
-    The possible options for this setting are the following:
+    A long-form log message that describes this event, optionally including expansions of fields and parameters so that the value is tailored to the event at hand.
+    It includes the `dest` (destination) attribute that defines how the event is treated, including the following options:
```
Comment (emarsh1995) [inline docs/modules/operation/pages/deep-dive/events/event-definition.adoc:31]:
```suggestion
    It includes the `dest` (destination) attribute, which defines how the event is treated, including the following options:
```

File: docs/modules/operation/pages/deep-dive/events/event-definition.adoc [hunk 2]
```diff
@@ -27,8 +27,8 @@ Description::
...
     logndisplay:::
         Both log the event in the database and display it in the Web UI.
```
Comment (emarsh1995) [inline docs/modules/operation/pages/deep-dive/events/event-definition.adoc:33]:
```suggestion
        Log the event in the database and display it in the web UI.
```

File: docs/modules/reference/pages/glossary.adoc [hunk 3]
```diff
@@ -139,7 +139,7 @@ Minions communicate with these remote devices while OpenNMS performs coordinatio
 Nephron:: An OpenNMS component that enables horizontal scaling of flow processing.
 Not required for flow processing, but can improve performance for deployments with a very high volume of flow documents.
 
-Network Operating Center (NOC):: One or more locations where a network is monitored and controlled.
+Network Operations Center (NOC):: One or more locations where a network is monitored and controlled.
 (See https://en.wikipedia.org/wiki/Network_operations_center[Network Operations Center].)
```
Comment (emarsh1995) [inline docs/modules/reference/pages/glossary.adoc:143]:
```suggestion
Network Operations Center (NOC):: One or more locations where a network is monitored and controlled (see https://en.wikipedia.org/wiki/Network_operations_center[Network Operations Center]).
```

**Review by emarsh1995 — CHANGES_REQUESTED** [review]
A couple changes. :)

**Review by emarsh1995 — APPROVED** [review]
LGTM! Thanks. :)

**Pull request merged**